manifests/keystone/endpoint.pp

```diff
@@ -32,6 +37,11 @@
     include ::ntnuopenstack::swift::endpoint
   }
 
+  # If there is a password for barbican in hiera, define an endpoint for swift.
```

Comment:
Denne kommentaren skaper mye mening :)

Reply:
Haha! Nei, den der skapte ikke så mye mening. Var vel litt kopipasta ute og gikk da serru. Men hadde du to kommentarer sa du? Jeg ser bare en?